src/assets/data/faq/index.js

```diff
@@ -174,7 +174,7 @@ const data = {
       category: "Meshery",
       answer: [
         "Meshery is an open source, vendor neutral projects that facilitates testing across meshes.",
-        "Other performance benchmark tools are not packaged into a mesh testing utility, butare only a load-generators unto their own.",
+        "Other performance benchmark tools are not packaged into a mesh testing utility, but are only a load-generators unto their own.", 
```

Comment:
I'm not able to digest this part : `but are only a load-generators unto their own.`

Reply:
@warunicorn19 Can you help me with this?

Reply:
i don't see anything wrong with the line except for the `a` in there.

Reply:
```suggestion
        "Other performance benchmark tools are not packaged into a mesh testing utility, but are only load-generators unto their own.", 
```

Reply:
ohh alright! I think my lack of knowledge on the topic.

Reply:
```suggestion
        "Other performance benchmark tools are not packaged into a mesh testing utility, but are only load-generators unto their own.", 
```